package.json

```diff
@@ -69,12 +69,12 @@
     "sane": "^1.1.2",
     "semver": "^4.3.6",
     "source-map": "0.1.31",
-    "stacktrace-parser": "frantic/stacktrace-parser#493c5e5638",
+    "stacktrace-parser": "^0.1.3",
     "uglify-js": "2.4.16",
     "underscore": "1.7.0",
     "wordwrap": "^1.0.0",
     "worker-farm": "^1.3.1",
-    "ws": "^0.7.2",
+    "ws": "^0.8.0",
```

Comment:
I'm trying to get `npm shrinkwrap`, but lets keep the versions pinned in the mean time.